Makefile

```diff
@@ -9,7 +9,8 @@ STYLES_DIR ?= resources/themes
 CACHE_DIR ?= cache
 
 STYLE ?= owncloud
-VERSION ?= 10.1.0
+VERSION ?= 10.0.10
```

Comment:
FYI in just merged https://github.com/owncloud/docs/pull/560 you set
`VERSION ?= 10.1.0`
which you would now "revert" back in case you merge this PR...

Reply:
That was merged by @PVince81 for the next release. I'm currently discussing, in #documentation, about the branching strategy, going forward. Thanks for commenting on this.

Reply:
@mmattel, reverted.